src/main/java/hudson/plugins/analysis/collector/AnalysisGraphConfiguration.java

```diff
@@ -77,7 +77,7 @@ public boolean getCanDeacticateOtherTrendGraphs() {
 
     @Override
     public String serializeToString() {
-        return super.serializeToString() + SEPARATOR + serializeBoolean(canDeactivateGraphs);
+        return super.serializeToString() + serializeBoolean(canDeactivateGraphs);
```

Comment:
I did it here because there was a test failing because of this just after upgrade analysis-core version, which is related to this PR. IMO we should fix broken tests derived from changes in the current PR.